### PR TITLE
Suppress YDB queue handling during YDB-to-GRDB migration.

### PR DESCRIPTION
**Pull request by charlesmchen-signal**

### CODE REVIEW

File: YapDatabase/YapDatabaseConnection.h [hunk 1]
```diff
@@ -294,6 +294,8 @@ typedef NS_OPTIONS(NSUInteger, YapDatabaseConnectionFlushMemoryFlags) {
 **/
 @property (atomic, assign, readonly) uint64_t pendingTransactionCount;
 
+@property (atomic) BOOL ignoreQueues;
```
Comment (charlesmchen-signal) [inline YapDatabase/YapDatabaseConnection.h:297]: By setting this flag we can use GRDB transactions within YDB transactions.

**Comment by charlesmchen-signal** [comment]
See: https://github.com/signalapp/Signal-iOS-Private/pull/1329

**Review by Imperiopolis-Signal — APPROVED** [review]

**Comment by charlesmchen-signal** [comment]
PTAL @michaelkirk-signal

**Review by michaelkirk-signal — APPROVED** [review]
LGTM

**Pull request merged**